### DISCUSSION TITLE
Did Albert Einstein lead physicists and cosmologists on a 100-year delusion?

### DISCUSSION THREAD

**Discussion by Savorinen** [a-cosmology-group/acg#134] [Cosmology Models]
Did Albert Einstein lead physicists and cosmologists on a 100-year delusion?

In space, expanding lights interact with each other, accelerating each other’s expansion, and thus the pace of expanding lights accelerates in proportion to the expansion of matter and light.

And this has already been scientifically proven.

And you, too, would accept it if Einstein had not snatched from his hat a new god named Curved Space.

Now, however, you believe in the existence of that god, a curved space. And the culprit for this is Einstein. And you yourself.

It is thought that Einstein would have presented the same as me, that is, that the nuclei of atoms expand and recycle a dark expanding pushing force that has e.g. the nature of the expanding light.

He would have said that, for example, the trajectory of expanding light protruding past the Sun bends towards the Sun, because billions of years of space-expanded light that protrudes toward the Sun contains a pushing force that pushes photons protruding past the Sun toward the Sun.

Einstein would have predicted that during the Eclipse, this could be proved by observing the stars behind the Sun whose location is known.

And then the matter would have been scientifically proven.

And no one, ever after that, would have thought of making an idiotic argument about the hokkus puck of space that supposedly curves, expands, waves, twists, stretches and bangs always according to what is needed to explain someone’s observation.

Only losers explain observations with hokkus pokkus space because they are unable to explain observations using space-moving and space-changing systems.

Hokkus pokkus space makes it easy to give features that allow hokkus pokkus space to do it and this always according to what ever needs to be explained without really explaining anything at all.

The expanding space is the emperor naked

😃

The expanding space is a buzz

The curving space is a hoax

The undulating space is a buzz

The orbiting space is a buzz

None of these hokkus kick spaces are needed.

It is enough that the lights consist for the most part of us dark expanding waves which, when interacting with each other, accelerate each other’s expansion and thus cause each other’s momentum to accelerate in the same proportion as matter and light expand.

Once this is understood and outlined, one understands that such hokkus pokkus spaces are not needed.

At the same time, many things that have remained mysteries so far are explained without unexplained dark matter, dark energies, driving forces, and so on.

As well as

1. The cosmological redshift of light

2. The so-called gravitational lens phenomenon

3. The so-called gravitational redshift

4. Double slit test

5. How lights can always move at the pace characteristic of light.

Jukka Savorinen

🙂

**Comment by RedshiftDrift** [comment]
> In space, expanding lights interact with each other, accelerating each other’s expansion, and thus the pace of expanding lights accelerates in proportion to the expansion of matter and light.  And this has already been scientifically proven.

No it has not.

**Reply by Savorinen** [reply]
The evidence for that already exists. However, these evidences are erroneously used to prove that space itself is expanding. And that space itself is curved.

Build a long chute far from the Earth into space and place a telescope on the other end of it to observe a distant galaxy whose location we know. You will notice that that distant galaxy seems to be in a different place than we know it to be.

**Reply by HanDeBruijn** [reply]
Discussion closed:  [General Relativity, blessing or curse?](https://github.com/orgs/a-cosmology-group/discussions/78)

**Reply by Savorinen** [reply]
"Discussion closed"

?🤔

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
*“Discussion is impossible with someone who claims not to seek the truth, but already to possess it.”*
― Romain Rolland, [Above The Battle](https://www.goodreads.com/work/quotes/2145562)

@Savorinen 
Optical aberration is not evidence for expanding or curved space.  Sound waves exhibit aberration in Euclidean space.
Additionally, science does not "prove" anything.
